subsys/logging/log_backend_rtt.c

```diff
@@ -201,7 +201,8 @@ static void on_write(int retry_cnt)
 static int data_out_block_mode(uint8_t *data, size_t length, void *ctx)
 {
 	int ret = 0;
-	int retry_cnt = CONFIG_LOG_BACKEND_RTT_RETRY_CNT;
+	int retry_cnt = IS_ENABLED(CONFIG_LOG_BACKEND_RTT_MODE_BLOCK) ?
```

Comment:
can you add a comment which would explain that this function is also used in drop mode if operations are synchronous?

Reply:
done